CHANGELOG.md

```diff
@@ -6,134 +6,138 @@ All notable changes to the "prettier-vscode" extension will be documented in thi
 
 ## [Unreleased]
 
+## [1.3.4]
```

Comment:
It certainly won't be version 1.3.4

Let it under [unreleased], a script does the rest :smiley: